README.md

```diff
@@ -1 +1,1178 @@
+
+![Honeycomb](images/ballerina-honeycomb.svg "Ballerina-Honeycomb")
+
+- **Make Requests** : To perform actions on student  management service, a console based client program has been written in Ballerina for your ease of making requests.
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- [Docker](https://docs.docker.com/engine/installation/)
+- [MYSQL](https://github.com/Shairam/Ballerina-Honeycomb/blob/master/resources/testdb.sql)
+- A Text Editor or an IDE 
+
+## Implementation
+
+> If you want to skip the basics, you can download the GitHub repo and continue from the "Testing" section.
+
+### Implementing database
+ - Start MYSQL server in your local machine.
+ - Create a database with name `testdb` in your MYSQL localhost. If you want to skip the database implementation, then directly import the [testdb.sql](https://github.com/Shairam/Ballerina-Honeycomb/blob/master/resources/testdb.sql) file into your localhost. You can find it in the Github repo.
+ 
+ 
+ 
+
+### Create the project structure
+        
+ For the purpose of this guide, let's use the following package structure.
+        
+    
+    ballerina-honeycomb
+           └── guide
+                ├── students
+                │   ├── student_management_service.bal
+                │   ├── student_marks_management_service.bal
+                ├── client_service
+                |         └── client_main.bal
+                └── ballerina.conf
+        
+
+- Create the above directories in your local machine, along with the empty `.bal` files.
+
+- You have to add the following lines in your [ballerina.conf](https://github.com/Shairam/Ballerina-Honeycomb/blob/master/ballerina.conf) to send the service traces to Honeycomb in Zipkin format using Opentracing.
+
+```ballerina
+[b7a.observability.tracing]
+enabled=true
+name="zipkin"
+
+[b7a.observability.tracing.zipkin]
+reporter.hostname="localhost"
+reporter.port=9411
+
+# Send the spans in V1 format as honeycomb-opentracing-proxy supports on V1
+reporter.api.context="/api/v1/spans"
+reporter.api.version="v1"
+
+
+reporter.compression.enabled=false
+```
+
+- Then open the terminal and navigate to `ballerina-honeycomb/guide` and run Ballerina project initializing toolkit.
+
+``
+   $ ballerina init
+``
+- Also you need to clone and build the ballerina-zipkin-extension in the following repository [https://github.com/ballerina-platform/ballerina-observability/tree/master/tracing-extensions/modules.](https://github.com/ballerina-platform/ballerina-observability/tree/master/tracing-extensions/modules) 
+
+- After building  move to `ballerina-zipkin-extension>/target/distribution/` and copy all the jar files to your `bre/lib` folder in your ballerina distribution.
+### Development of student and marks service with Honeycomb
+
+Now let us look into the implementation of the student management with observability.
```

Comment:
Should it be "the student management service..."?

Reply:
Yes, noted. I will update them.